junit-runtime/src/main/scala/org/hamcrest/BaseDescription.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Ported from https://github.com/hamcrest/JavaHamcrest/
+ */
+package org.hamcrest
+
+import java.util.Arrays
+import java.lang.String.valueOf
```

Comment:
I'd rather you use `String.valueOf` explicitly when you need it. I was wondering what the bar `valueOf()` was.